NitroxServer/GameLogic/Bases/BaseData.cs

```diff
@@ -11,36 +11,11 @@ namespace NitroxServer.GameLogic.Bases
     public class BaseData
     {
         public const long VERSION = 1;
-        [ProtoMember(1)]
-        public Dictionary<NitroxId, BasePiece> SerializableBasePiecesById
-        {
-            get
-            {
-                lock (basePiecesById)
-                {
-                    return new Dictionary<NitroxId, BasePiece>(basePiecesById);
-                }
-            }
-            set { basePiecesById = value; }
-        }
 
-        [ProtoMember(2)]
-        public List<BasePiece> SerializableCompletedBasePieceHistory
-        {
-            get
-            {
-                lock (completedBasePieceHistory)
```

Comment:
Is it safe to remove the locks?  All of the functions below leverage these locks too.  For example, I believe AddBasePiece can be called from individual client threads while the saving can happen on the main thread based on a timer (at the same time)

Reply:
It "MIGHT" run into issues depending on how protobuf-net is programmed but It would appear Protobuf-net is thread safe

Reply:
https://github.com/protobuf-net/protobuf-net/wiki/Capabilities